File: cli/src/app/monlog.ts
import { enums } from './enums';

export namespace monlog { 
	/* monlog -  Логирование */ 

 export interface   logcall { // Сообщения
	logcallId:string; // Primary key field
	ID_BD:string; //Устройство -> MONDEV_BDEVICES
	DCALL:string;  // Дата опроса
	AType:string; //Тип архива -> MOND_ATYPE
	CPORT:string; // Порт
	DURATION:Number; // Длительность
	CRESULT:string; // Результат
	// add dereference fields 
	ID_BD_name :string; // dereference for MONDEV_BDEVICES
	AType_name :string; // dereference for MOND_ATYPE
 }
}

File: .net/Data.NET/monlog.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace a_srv.models { 
	/* monlog -  Логирование */ 

 public class  logcall { // Сообщения
	 public System.Guid  logcallId{ get; set; } // Primary key field
	public System.Guid  ID_BD { get; set; } //Устройство
	[Required]
	public DateTime  DCALL{ get; set; } // Дата опроса
	public System.Guid  AType { get; set; } //Тип архива
	public string  CPORT{ get; set; } // Порт
	public double?  DURATION{ get; set; } // Длительность
	public string  CRESULT{ get; set; } // Результат
 }
}